### ISSUE TITLE
Izzyondroid update

### ISSUE THREAD

**Issue by AkaneTan** [FoedusProgramme/Gramophone#18]
@IzzySoft hi sorry for the ping, but the app on izzydroid is outdated XD
Can I expect an update soon?
P.S. I saw the fastlane thing for songsync, is it also something necessary for Gramophone?

**Comment by IzzySoft** [comment]
> the app on izzydroid is outdated XD

No, it's on the latest release There was only one release up to now: 0.0.1. The other one is marked pre-release and thus was not picked up. Do you want me to pick up pre-releases as well? It'd probably be better if you mark those ones you wish to have distributed as stable I'd say, but that's of course up to you.

> I saw the fastlane thing for songsync, is it also something necessary for Gramophone?

That would be nice indeed, as it would give you control how your app is presented – without having to ask me to update things, as Fastlane would be picked up along with any new release. You can use my [Fastlane Cheat Sheet](https://gitlab.com/-/snippets/1895688) for guidance with the setup – and ask me should you need help. Let me know when it's ready, and I enable it for pull on my end.

**Comment by AkaneTan** [comment]
@IzzySoft nah there's a 1.0.1 release however GitHub is not showing that properly.
https://github.com/AkaneTan/Gramophone/releases/tag/1.0.1

**Comment by IzzySoft** [comment]
Ah, that crazy Github bug again! That release doesn't show up via the API either then, which is why my updater cannot find it. You probably created that release, then deleted it and re-created it with the same name? That's what I know to trigger that bug. If you should run into a situation where you need to delete and recreate a release, make sure to name the new tag/release slightly different (here e.g. `1.0.1.0` would do), just as a hint. Will pick that one manually now (should show up with the next sync around 6 pm UTC then). Thanks for pointing to it!

**Comment by AkaneTan** [comment]
@IzzySoft Btw is izzydroid picking up apps automatically or submitted by users? Cuz to my surprise Gramophone is added to library when there's almost no stars

**Comment by IzzySoft** [comment]
Depends on. Inclusion can be requested/suggested, but most apps are found by one of my crawlers and suggested to me directly.

![image](https://github.com/AkaneTan/Gramophone/assets/6781438/94eab268-985f-4fe6-82c9-41f0a033e8df)

**Comment by opusforlife2** [comment]
If it's on the Izzy repo, does that mean you eventually intend to submit this to F-Droid?

**Comment by AkaneTan** [comment]
> If it's on the Izzy repo, does that mean you eventually intend to submit this to F-Droid?

I'm lazy to do fastlane but if somebody did a pr for fastlane then yes

**Comment by IzzySoft** [comment]
> if somebody did a pr for fastlane

There you go :smile: That's what is set up for my repo here.

**Comment by AkaneTan** [comment]
@IzzySoft Hi I changed fastlane's app logo month ago but didn't see it's changing on izzyondroid, can you confirm if it's being fetched correctly?

**Comment by IzzySoft** [comment]
Let me check what's set up – and what you might mean by "Logo" (icon? featureGraphic?):

```yaml
    Fastlane: https://github.com/AkaneTan/Gramophone/tree/beta/fastlane/metadata/android
    FastlaneFeatures: shortdesc,fulldesc,icon,screenshots
```

Seems to match. Last release: 2023-09-22. Month ago: 2023-10-21. Well, that explains: no new release since, so nothing picked up. Fastlane is checked whenever a new release is pulled – as that's the point one can expect the two match. If Fastlane would be checked daily, it might fetch adjustments to it which are just "in preparation of" the not-yet-happened release. :man_shrugging:

**TL;DR:** Works as designed, yes :zany_face:

**Comment by IzzySoft** [comment]
Sorry to abuse this issue, @AkaneTan – but could you please fix this:

    en-US/short_description.txt has 111 chars

Fastlane specs say max 80 chars. Maybe you simply strip the part of "strictly following android standards"? That's the least relevant part for non-techy folks using the app :wink: Thanks!

**Comment by nift4** [comment]
https://github.com/AkaneTan/Gramophone/commit/84d812837ba9910b75e46c6b1495bb340edae9c8
this wasn't enough?

**Comment by nift4** [comment]
ah nevermind its not in a release yet that's why - it'll be fixed in the next one then

**Comment by IzzySoft** [comment]
Ah, good! Then it should be fixed when the next release is being pulled, all fine then!
